pkg/kubectl/cmd/config/view.go

```diff
@@ -106,6 +109,11 @@ func (o ViewOptions) Run(out io.Writer, printer kubectl.ResourcePrinter) error {
 		}
 	}
 
+	if o.Short {
+		fmt.Printf("short\n")
```

Comment:
Actually it wasn't intentional, just debugging cruft. If we want to make `--short` incompatible with `-o template`, that should be done explicitly.